devtools/x/src/diff_summary.rs

```diff
@@ -14,6 +14,9 @@ pub struct Args {
     #[structopt(name = "COMPARE_SUMMARY")]
     /// Path to the comparison summary
     compare_summary: PathBuf,
+    #[structopt(name = "OUTPUT_FORMAT")]
+    /// output can be json, toml, or text
```

Comment:
This doesn't match what's implemented (no TOML support, and "text" isn't recognized. Please also add a default value of text).

Reply:
done!

<img width="1052" alt="Screen Shot 2020-12-09 at 5 06 25 PM" src="https://user-images.githubusercontent.com/1316043/101707829-dfae6b80-3a40-11eb-8402-a52abf20ef15.png">